Jekyll build is broken

See https://github.com/fortran-lang/fortran-lang.org/actions/runs/431313773

```
Run cd src
  cd src
  sudo gem install bundler
  bundle config path .bundle
  bundle install
  shell: /bin/bash -e {0}
  env:
    SRC_DIR: src
    PUBLISH_DIR: gh-pages
    SITE_URL: https://fortran-lang.org
    FPM_INDEX: https://raw.githubusercontent.com/fortran-lang/fpm-registry/master/index.json
Successfully installed bundler-2.2.2
Parsing documentation for bundler-2.2.2
Installing ri documentation for bundler-2.2.2
Done installing documentation for bundler after 3 seconds
1 gem installed
/usr/lib/ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
	from /usr/lib/ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
	from /usr/local/bin/bundle:23:in `<main>'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/buildPRPreview.yml
+++ b/.github/workflows/buildPRPreview.yml
@@ -38,6 +38,11 @@ jobs:
         path: ${{env.PUBLISH_DIR}}
         ref: 'gh-pages' 
     
+    # Install ruby 2.7.x
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+
     # Setup Jekyll in SRC_DIR
     - name: Setup Jekyll
       run: |

--- a/.github/workflows/buildSite.yml
+++ b/.github/workflows/buildSite.yml
@@ -32,6 +32,11 @@ jobs:
       with:
         path: ${{env.PUBLISH_DIR}}
         ref: 'gh-pages'
+    
+    # Install ruby 2.7.x
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
 
     # Setup Jekyll in SRC_DIR
     - name: Setup Jekyll

